### ISSUE TITLE
Exception when event does not have a pull request number

### ISSUE THREAD

**Issue by ssbarnea** [jesusvasquez333/verify-pr-label-action#19]
The code should expect to get an empty string and fallback to previous PR lookup logic instead of throwing a stacktrace.

```
 Traceback (most recent call last):
  File "/verify_pr_lables.py", line 64, in <module>
    pr_number=int(pr_number_str)
ValueError: invalid literal for int() with base 10: ''
Valid labels are: bug, enhancement, feature, skip-changelog, deprecated, major
A valid pull request number input must be defined when triggering on "pull_request_target". The pull request number passed was "".
```

Details: https://github.com/ansible-community/molecule/pull/2876/checks?check_run_id=1241412708

**Comment by jesusvasquez333** [comment]
Thank for catching this issue @ssbarnea.

Indeed, the situation when the pull request number is not defined should be handled, instead of generating the traceback. 

However, when the trigger event is `pull_request_target` like in this example, the pull request number can not be extracted from the environmental variables.

**Comment by jesusvasquez333** [comment]
The fix for this issue is available in version [v1.4.0](https://github.com/jesusvasquez333/verify-pr-label-action/releases/tag/v1.4.0).
